docs/api_summary.rst

```diff
@@ -0,0 +1,28 @@
+
+===========
+API Summary
+===========
+
+Summary of public functions and classes exposed
+in *onnxmltools*.
+
+.. contents::
+    :local:
+
+Converters
+==========
+
```

Comment:
In the live website, I didn't see the details for the keras or sklearn convert functions.